data/scopes/yealink-T30.ini

```
@@ -0,0 +1,51 @@
+[metadata]
+scopeType = "model"
+displayName = "Yealink T30/P"
+version = 4
+
+[data]
+cap_contrast = ""
+contrast = 
+cap_brightness = ""
+brightness =
+cap_screensaver_file = ""
+cap_screensaver_time_blacklist = "3,5,7,10,1200,2400,3000,3600"
+screensaver_time = 600
+cap_background_file = ""
+cap_backlight_time_blacklist = "0,3,5,7,10,15,30,60,120,300,600,1200,1800,2400,3000,3600"
+backlight_time = 0
+cap_ldap_tls_blacklist =
+provisioning_url_filename = 
+tmpl_phone = "yealink.tmpl"
+cap_linekey_count = 0
+cap_linekey_type_blacklist = ""
+cap_softkey_count = 4
+cap_softkey_type_blacklist = ""
+cap_expkey_count = 0
+cap_expkey_type_blacklist = 
+cap_ringtone_count = 4
```

are you sure that only 4 ring tones are supported?
```suggestion
cap_ringtone_count = 10
```

---

This phone has only 4 ring type